docs/source/quickstart.rst

```diff
@@ -1,88 +1,110 @@
 .. _quickstart:
 
-Quickstart
-==========
+Quickstart: Install MLflow, instrument code & view results in minutes
+======================================================================
 
-Installing MLflow
+
+In 15-20 minutes, you will:
+
+- Install MLflow
+- Add MLflow tracking to your code
+- View run metrics in the MLflow tracking UI
+- (Optional) Run a tracking server to share results with others
+- (Optional) Use Databricks to store your results
+- Store the models produced by your runs
+- Run a model from a previous run
+
+As a data scientist, your explorations involve running your evolving training code many times. MLflow Tracking allows you to record important information your run, review and compare it with other runs, and share results with others. As an ML Engineer or MLOps professional, it allows you to compare, share, and deploy the best models produced by the team.
```

Comment:
I love that we call out `As an ML Engineer or MLOps professional, it allows you to compare, share, and deploy the best models produced by the team.` at the end, but the quickstart doesn't cover these topics. We should definitely follow up with quickstart content for ML Engineers / MLOps. Just leaving this here as a note, no immediate action item